src/App.js

```diff
@@ -66,13 +63,11 @@ export default function App() {
 
   function handleLikePlace(name) {
     let newArray
```

Comment:
maybe you can name the array a little bit more descriptive. so it's easier for a third person to read and understand the reason of the "newArray"

Reply:
Good point! Thank you! 👍🏻